python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -364,6 +367,12 @@ class RandomForestClassifier(Base):
         self.n_streams = handle.getNumInternalStreams()
         self.seed = seed
 
+        if ((seed != -1) and (n_streams != 1)):
+            warnings.warn("n_stream should be equal to 1 while setting the "
+                          "seed. If n_streams!=1 then setting the seed will"
+                          "not help in making the result reproducible over"
+                          " multiple runs")
```

Comment:
see message change in the other one I suggested

Reply:
Hey @Salonijain27 @dantegd Why do we need this PR ? Random seed for multi-stream will be fixed in #1281